src/main/java/de/tum/in/www1/artemis/service/dto/athena/TextFeedbackDTO.java

```diff
@@ -26,31 +24,7 @@ public record TextFeedbackDTO(long id, long exerciseId, long submissionId, Strin
     public static TextFeedbackDTO of(long exerciseId, long submissionId, @NotNull Feedback feedback, TextBlock feedbackBlock) {
         Integer startIndex = feedbackBlock == null ? null : feedbackBlock.getStartIndex();
         Integer endIndex = feedbackBlock == null ? null : feedbackBlock.getEndIndex();
-        var gradingInstructionId = feedback.getGradingInstruction() == null ? null : feedback.getGradingInstruction().getId();
-        return new TextFeedbackDTO(feedback.getId(), exerciseId, submissionId, feedback.getText(), feedback.getDetailText(), feedback.getCredits(), gradingInstructionId,
-                startIndex, endIndex);
-    }
-
-    /**
-     * Creates a TextBlockRef (feedback + text block combined) from this DTO and a TextSubmission
-     */
-    public TextBlockRef toTextBlockRef(TextSubmission onSubmission, GradingInstruction gradingInstruction) {
```

Comment:
Comment for reviewers: The logic for this now lives in the client (`text-submission-assessment.component.ts`, `loadFeedbackSuggestions`).